lib/hanami/utils/kernel.rb

```diff
@@ -404,15 +413,14 @@ def self.Integer(arg)
       #   # Missing #respond_to?
       #   input = BasicObject.new
       #   Hanami::Utils::Kernel.BigDecimal(input) # => TypeError
-      def self.BigDecimal(arg)
+      def self.BigDecimal(arg, precision: BIGDECIMAL_DEFAULT_PRECISION)
```

Comment:
Running my general "if/else" vs. "case/when" benchmark "if/else" is still two times faster, even with Jruby. But of course this boils down to number of Procs are being called..